src/components/screen/Setting/ChangePwView.tsx

```diff
@@ -0,0 +1,96 @@
+import { Alert, KeyboardAvoidingView, SafeAreaView } from 'react-native';
+import {
+  DefaultTheme, withTheme,
+} from 'styled-components/native';
+import { InnerContainer, ModalCloseButton, ModalHeader, ModalTitle, StyledTextInput } from './styles';
+import React, { useState } from 'react';
+
+import Button from '../../shared/Button';
+import { Ionicons } from '@expo/vector-icons';
+import { getString } from '../../../../STRINGS';
+
+export interface Props {
+    theme: DefaultTheme;
+    close(): void;
```

Comment:
I don't think this is a good idea if this is a `screen` component. It'd be better to pass the param to `react-navigation` and get it from the `route` param.